### PR TITLE
updating actions page

### PR DESCRIPTION
**Pull request by amit9192**

### CODE REVIEW

File: docs/Learn/Actions.md [hunk 1]
```diff
@@ -2,11 +2,11 @@
 
 To allow our users to close the troubleshooting loop through Komodor, we’re adding the ability to perform Actions through the platform.
 
-Using Komodor you can run multiple actions against your resources, including the ability to terminate Pod/s, scale your Deployments, re-trigger your Jobs, revert a deployment and many more to come.
+Using Komodor you can run multiple actions against your resources, you'll be able to easily track what was done and by whom.
```
Comment (michaeldeift) [inline docs/Learn/Actions.md:5]: Change it to two sentences

File: .vscode/dryrun.log [hunk 2]
```diff
@@ -0,0 +1,5 @@
+make: Entering directory `/Users/amitbaroz/Documents/docs'
```
Comment (michaeldeift) [inline .vscode/dryrun.log:1]: Do you want to commit all these vscode configs?

File: docs/Learn/Actions.md (hunk 1 above)
Comment (amit9192) [reply docs/Learn/Actions.md]: fixed

File: .vscode/dryrun.log (hunk 2 above)
Comment (amit9192) [reply .vscode/dryrun.log]: fixed

**Review by michaeldeift — APPROVED** [review]

**Pull request merged**